charts/opentelemetry-collector/values.yaml

```diff
@@ -285,6 +285,23 @@ ingress:
 #      hosts:
 #        - collector.example.com
 
+# Additional ingresses - only created if ingress.enabled is true
+# Useful for when differently annotated ingress services are required
+extraIngresses: []
```

Comment:
Can we move it under `ingress->additionalIngresses`?

Reply:
Moved to `ingress.additionalIngresses`.